### PR TITLE
Add winget package support

### PR DESCRIPTION
**Pull request by rubo**
## Changes

- Updated the release workflow to submit a winget package update
- Updated the release workflow to upload package artifacts for manual verification if needed before publishing to GitHub Releases
- Moved PPA, Windows Package Manager, and Homebrew scripts to another workflow \
  Current flow:
    -  Build > Approve > GitHub, Docker, Downloads
    - GitHub > PPA (> Copy PPA), winget, Homebrew
- Updated the release workflow to publish to GitHub Releases immediately if a draft release exists; otherwise, drafts a new release
- Updated the README with winget instructions

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [x] Documentation update
- [x] Build-related changes
- [x] Other: Windows Package Manager (winget) support

#### Requires testing

- [x] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [x] No

## Remarks

The v1.17.0 is already available with `winget install nethermind`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ Release builds are available on the [Releases page](https://github.com/nethermin
 3. To run the launcher: `nethermind`
 4. To run the runner: `nethermind -c mainnet`
 
+#### On Windows using Windows Package Manager
+
+1. `winget install nethermind` (running as administrator recommended)
+2. To run the launcher: `nethermind.launcher.exe`
+3. To run the runner: `nethermind.runner.exe -c mainnet` \
+   Or simply `nethermind -c mainnet` if step 1 has been run as administrator
```
Comment (FalcoXYZ) [inline README.md:41]: Trying to run this but it won't let me. Did you try this already? 

```
PS C:\Users\Falco> nethermind -c mainnet
2023-03-02 14-50-14.8357|Nethermind starting initialization.
2023-03-02 14-50-14.8861|Loading embedded plugins
2023-03-02 14-50-14.8864|  Found plugin type Nethermind.Consensus.AuRa.AuRaPlugin
2023-03-02 14-50-14.8865|  Found plugin type Nethermind.Consensus.Clique.CliquePlugin
2023-03-02 14-50-14.8867|  Found plugin type Nethermind.Consensus.Ethash.EthashPlugin
2023-03-02 14-50-14.8868|  Found plugin type Nethermind.Consensus.Ethash.NethDevPlugin
2023-03-02 14-50-14.8869|  Found plugin type Nethermind.Hive.HivePlugin
2023-03-02 14-50-14.8870|  Found plugin type Nethermind.UPnP.Plugin.UPnPPlugin
Resolved executing directory as C:\Users\Falco\AppData\Local\Microsoft\WinGet\Links.
2023-03-02 14-50-14.9036|Plugin assemblies folder C:\Users\Falco\AppData\Local\Microsoft\WinGet\Links\plugins was not found. Skipping.
2023-03-02 14-50-14.9816|Loading standard NLog.config file from C:\Users\Falco\AppData\Local\Microsoft\WinGet\Links\NLog.config.
```

Installing as administrator also won't let me run `nethermind.launcher.exe` or `nethermind.runner.exe` anymore.

Comment (rubo) [reply README.md:41]: Everything works fine for me. Please DM me to discuss your issue.

**Comment by kamilchodola** [comment]
@rubo 
Installed nethermind with command provided and it is failing:
image

**Comment by kamilchodola** [comment]
@rubo 
UPDATE: installed once more as admin and works now - not sure if it is OK... Guess we will have reports with that.
nethermind.launcher.exe still not works for me

**Comment by rubo** [comment]
@kamilchodola More looks like you haven't restarted your shell after the installation as the installer suggested.

**Comment by kamilchodola** [comment]
@rubo 
Done that few times.
Only "nethermind" command works for me

**Comment by FalcoXYZ** [comment]
@kamilchodola please try: 

install without admin
close your shell fully and launch it again. 
try running: nethermind.runner.exe or nethermind.launcher.exe

**Review by FalcoXYZ — APPROVED** [review]
Tested several times, nethermind.launcher.exe and nethermind.runner.exe works fine.
Remove admin instructions as we already discussed though. 

LGTM!

**Pull request merged**